src/modbus_rtu.rs

```diff
@@ -305,7 +308,35 @@ impl ModbusRTU for Modbus {
     ///
     /// TODO: implement rtu_set_custom_rts()!
     fn rtu_set_custom_rts(&mut self, _mode: RequestToSendMode) -> Result<i32, Error> {
-        unimplemented!()
+        //I need to distinguish between RtuRtsUp and RtuRtsDown, and now it's for RtuRtsUp."
+        unsafe extern "C" fn custom_rts_callback(_ctx: *mut ffi::modbus_t, on: c_int) {
+            let mut chip = Chip::new("/dev/gpiochip0").expect("Failed to open gpiochip0");
```

Comment:
Not ideal that this gets reopened every time, can you pass this somehow via an handle?